recipes/netpbm/build.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+# cp config.mk.in config.mk
+# envsubst '${PREFIX}' < ${RECIPE_DIR}/config.template >> config.mk
+cp ${RECIPE_DIR}/config.mk config.mk
+make
+TMPDIR=`mktemp -d -u`
+make package pkgdir=${TMPDIR}
+sed -i s#/usr/bin/perl#/opt/anaconda1anaconda2anaconda3/bin/perl#g ${TMPDIR}/bin/*
```

Comment:
What does this do?

Reply:
Netpbm has some strange peculiarities in the install process. For making a package, the code installs the files to a `$TMPDIR`, so, to the conda build reconizes correctly the installed files, they have to be copied from the `$TMPDIR` to the `$PREFIX`.

Reply:
Why can't `$TMPDIR` be just `$PREFIX`?

Reply:
Making Netpbm directly to prefix causes a build error:

```
Directory $PREFIX already exists.  Please specify a 
directory that can be created fresh, like this: 
  make package pkgdir=/tmp/newnetpbm
```

Reply:
Can you change the line to `s#/usr/bin/perl#/usr/bin/env perl#g` ?

Reply:
Ok!